spec/presenters/keyword_presenter_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordPresenter do
+  describe '#keyword' do
+    context 'given the urls contain the keyword vpn' do
+      it 'has matching_adword_urls with 2 urls https://www.thetopvpn.com and https://www.nordvpn.com' do
+        ads_top_urls = ['https://www.thetopvpn.com', 'https://www.nordvpn.com', 'https://www.vnexpress.net']
+        keyword = Fabricate(:keyword, ads_top_urls: ads_top_urls)
+        filter_params = { adwords_url_contains: 'vpn' }
+        result = described_class.new(keyword, filter_params).matching_adword_urls
+
+        expect(result).to eq(['https://www.thetopvpn.com', 'https://www.nordvpn.com'])
+      end
+    end
+
+    context 'given the urls do not contain the keyword vpn' do
+      it 'returns matching_adword_urls as empty array' do
```

Comment:
```suggestion
      it 'is empty' do
```

If you agree with the renaming of `describe`, we can make this one much easier 🚀